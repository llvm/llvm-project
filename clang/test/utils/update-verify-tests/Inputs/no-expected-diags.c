// expected-no-diagnostics
void foo() {
    a = 2;
}
