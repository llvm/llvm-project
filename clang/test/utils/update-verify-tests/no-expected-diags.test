# RUN: cp %S/Inputs/no-expected-diags.c %t.c && not %clang_cc1 -verify %t.c 2>&1 | %update-verify-tests
# RUN: diff --strip-trailing-cr %S/Inputs/no-expected-diags.c.expected %t.c
# RUN: %clang_cc1 -verify %t.c

