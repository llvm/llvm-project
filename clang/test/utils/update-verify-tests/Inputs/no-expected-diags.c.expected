void foo() {
    // expected-error@+1{{use of undeclared identifier 'a'}}
    a = 2;
}
